tool.go

```diff
@@ -91,6 +91,10 @@ func main() {
 	cmdBuild.Flags().AddFlag(flagTags)
 	cmdBuild.Flags().AddFlag(flagLocalMap)
 	cmdBuild.Run = func(cmd *cobra.Command, args []string) {
+		if err := verifyAndUnsetGOARCH(); err != nil {
```

Comment:
Imho this should only be referenced once in the `compiler` package, not for every command.